Lib/test/test_asyncio/test_ssl.py

```diff
@@ -1410,10 +1411,27 @@ async def client(addr):
             except (BrokenPipeError, ConnectionResetError):
                 pass
 
-            await future
-
-            writer.close()
-            await self.wait_closed(writer)
+            # Make sure _SelectorSocketTransport enters the delayed write
```

Comment:
You shouldn't modify existing tests, instead add a new one based on the previous one.

Reply:
Right, done.